.yarnrc.yml

```diff
@@ -0,0 +1,7 @@
+nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@yarnpkg/plugin-workspace-tools.cjs
```

Comment:
A lot of Yarn functionality is in plugins. Added the workspace-tools one, which lets us to stuff like `yarn workspaces foreach run test`